hudi-client/hudi-spark-client/src/main/scala/org/apache/hudi/HoodieSparkUtils.scala

```diff
@@ -21,21 +21,24 @@ package org.apache.hudi
 import org.apache.avro.Schema
 import org.apache.avro.generic.GenericRecord
 import org.apache.hadoop.fs.{FileSystem, Path}
+import org.apache.hudi.client.utils.SparkRowSerDe
```

Comment:
Moved HoodiesparkUtils, SparkAdaptorSupport and SparkAdaptor from hudi-spark module to hudi-spark-client module since we wanted to access SparkAdaptor from within BuiltInKeygen.